lib/honeybadger/insights/oban.ex

```diff
@@ -0,0 +1,28 @@
+defmodule Honeybadger.Insights.Oban do
+  @moduledoc false
+
+  use Honeybadger.Insights.Base
+
+  @required_dependencies [Oban]
+
+  @telemetry_events [
+    "oban.job.stop",
+    "oban.job.exception"
+  ]
+
+  def extract_metadata(meta, _name) do
+    Map.take(meta, [
+      :args,
+      :attempt,
+      :id,
+      :memory,
+      :prefix,
+      :queue,
+      :queue_time,
+      :state,
+      :tags,
+      :time,
+      :worker
+    ])
+  end
```

Comment:
All fields that are defined on a job are undocumented and deprecated. They should be taken off the job instead. Some of these attributes don't exist, and others are actually from `measurements`:

```suggestion
  def extract_metadata(%{conf: conf, job: job, state: state}, _name) do
    job
    |> Map.take(~w(args attempt id queue tags worker)a)
    |> Map.put(:prefix, conf.prefix)
    |> Map.put(:state, state)
  end
```

Reply:
👍🏻 I think I pulled these out while inspecting the instrumentation payload, so I was unaware of what is deprecated.

@sorentwo You are emitting a gaggle of telemetry events in Oban. Are there any others that would be useful to include by default?